scripts/202306xx_populate_paymenttype_default/add_fixed_payment_type_default.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable */
```

Comment:
The number of forms with payment is currently less than 50.

But however, I need this exercise. Will ping you separately on how you would write such a batching job.

Reply:
The number of forms with payment does not really matter that much, rather is the total number of forms created since payment feature is launched, cuz if we just add _id filter, we need to know how many rows will be affected in the index scan.

Reply:
+1 on this! 

Just went to try filtering by _id> objectId something

```
db.getCollection("forms")
.find({ 
    '_id': {$gt: ObjectId('some Id')},
    "payments_field": { $exists: true }
    }).count()

```

But doesn't seem to have a performance gain

there are a few [articles](https://studio3t.com/knowledge-base/articles/optimize-mongodb-queries/) about optimizing query performance. Maybe we have to create an index to preliminary filter by id/date before running the payment_field find query

Reply:
A few experiments using `.explain('executionstats')` (find without count) in MongoDB staging cluster (229792 total forms)


`db.getCollection("forms").find({}).explain('executionStats')`

1993ms, 229792 returned, 229792 examined, 12 execution stages

`db.getCollection("forms").find({"payments_field": { $exists: true }}).explain('executionStats')` 

2136ms, 57 returned, 229792 examined, 13 execution stages

`db.getCollection("forms").find({'_id': {$gt: ObjectId('5d1fa36ca2f1698500af5fcd')},
    "payments_field": { $exists: true }}).explain("executionStats")`

2294ms, 57 returned, 198856 examined, 14 execution stages

`db.getCollection("forms").find({'_id': {$gt: ObjectId('5f1fa36ca2f1698500af5fcd')},
    "payments_field": { $exists: true }}).explain("executionStats")`

1048ms, 57 returned, 104709 examined, 14 execution stages


So @mayying is right! Just that we need to select the 'break point' properly else it will end up being a little slower due to more executions!